pymobiledevice3/services/crash_reports.py

```diff
@@ -18,6 +18,15 @@ def __init__(self, lockdown: LockdownClient):
         self.lockdown = lockdown
         self.afc = AfcService(lockdown, service_name=self.COPY_MOBILE_NAME)
 
+    def __enter__(self):
```

Comment:
make the commit message of this change shorter

Reply:
done